.github/workflows/ci.yaml

```diff
@@ -421,6 +421,43 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
 
+  deploy-demo-env:
+    name: Deploy updated images to the demo environment
+    runs-on: self-hosted
+    if: github.event_name == 'release' || (github.event_name == 'push' && github.ref_name == 'main') || github.event_name == 'workflow_dispatch'
+    env:
+      IMAGE_REPOSITORY: ghcr.io/${{ github.repository_owner }}/trento-web
+      IMAGE_TAG: "${{ (github.event_name == 'release' && github.event.release.tag_name) || (github.event_name == 'push' && github.ref_name == 'main' && 'rolling') || github.sha }}"
+    needs: build-and-push-container-images
+    steps:
+      - name: Start a local k8s cluster
```

Comment:
Are these initial 2 steps needed? I would expect that these utilities are already in our server. Anyway, if they are not harmful go ahead

Reply:
They are not needed, but I'd keep then as they are useful in the case when we need to deploy a new demo VM where to deploy to. It removes the requirement of preinstalling anything on the server while at the same time we are being more declarative. wdyt?